if not os.path.exists(path):

```diff
@@ -877,7 +877,7 @@ def map_filename(filename):
             return (digits, prefix)
         
         # Setup custom path or default
-        if output_path.strip() != '':
+        if output_path.strip() == '':  
```

Comment:
This code is leftover from me using the WAS Node Suite as a template for making my nodes.  The purpose of the `!=` is to check for an empty path and, if **not** empty check for existing paths.   This change would make the path check happen ONLY if the path given is empty, which would not help except to skip the existing path check.